app/styles/_mixins.scss

```diff
@@ -0,0 +1,39 @@
+//Font Mixins
+@mixin font() {
+  font-family: $default-font;
+}
+
+@mixin headerFont() {
+  font-family: $header-font;
+}
+
+@mixin formElement() {
```

Comment:
Not sure if these mixins should be `header-font()` and `form-element()` to be consistent w/ `hidpi-background-image()`.